Jetcaster/mobile/src/main/java/com/example/jetcaster/ui/home/HomeViewModel.kt

```diff
@@ -70,6 +73,9 @@ class HomeViewModel @Inject constructor(
     // Holds the view state if the UI is refreshing for new data
     private val refreshing = MutableStateFlow(false)
 
+    private val subscribedPodcasts = podcastStore.followedPodcastsSortedByLastEpisode(limit = 10)
```

Comment:
NIT: I'd add a comment why this uses `shareIn` operator as when someone reads it in the future, they will think "this is unnecessary" and remove it :)